plugins/org.komodo.shell/src/org/komodo/shell/DefaultKomodoShell.java

```diff
@@ -538,5 +538,18 @@ private void writeCommandToRecordingFile( ShellCommand command ) {
     public String getShellPropertiesFile() {
         return PROPERTIES_FILE_NAME;
     }
+    
+    private String getVersion(){
+    	
+    	Properties properties = new Properties();
+    	try {
+    		ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+			properties.load(classLoader.getResourceAsStream("version.properties"));
+		} catch (IOException e) {
+			displayMessage( "Error during loading Komodo version " + e.getLocalizedMessage() );
+		}
+    	
+    	return properties.getProperty("komodo.version");
```

Comment:
If an exception occurs, the return value will be `null` and that will be printed out in the welcome message. I think it would be better to print out localized string like "Error obtaining version" or something like that.